src/bootstrap.jl

```diff
@@ -2,7 +2,8 @@
 # dynamically in future.
 
 struct RecoveredTBasket
-    data
+    data::Vector{UInt8}
+    offsets::Vector{UInt32}
```

Comment:
this looks like the underlying data for a 1-jagged array?

Reply:
that can be jagged or not, similar to the usual basket. If `offsets` is non-empty, it's jagged